packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -262,14 +262,28 @@ contract TaikoL1 is
         virtual
         returns (TaikoData.Config memory)
     {
+        // All hard-coded configurations:
+        // - treasury: 0xdf09A0afD09a63fb04ab3573922437e1e637dE8b
+        // - blockMaxTxs: 80 (limited by the PSE zkEVM circuits)
+        // - anchorGasLimit: 250_000 (based on internal devnet, its ~220_000
+        // after 256 L2 blocks)
         return TaikoData.Config({
-            chainId: 167_007,
-            blockMaxProposals: 403_200,
-            blockRingBufferSize: 403_210,
+            chainId: 167_008,
+            // Assume the block time is 3s, the protocol will allow ~1 month of
+            // new blocks
+            // without any verification.
+            blockMaxProposals: 864_000,
+            blockRingBufferSize: 864_010,
+            // Can be overridden by the tier config.
             maxBlocksToVerifyPerProposal: 10,
+            // Limited by the PSE zkEVM circuits.
```

Comment:
Because people complain about it and there is a path for the PSE circuits to be able to support arbitrary block sizes, maybe we can just set this to 15M gas?

Reply:
Ah good news! Changed to 15M.